src/Controller/Admin/TargetingController.php

```diff
@@ -65,7 +63,7 @@ public function ruleListAction(Request $request): JsonResponse
             ];
         }
 
-        return $this->jsonResponse($targets);
+        return $this->adminJson($targets);
```

Comment:
TargetingController does not extend PageController so we need to use JsonHelperTrait here